src/pages/components/notification/code.mdx

```diff
@@ -4,66 +4,132 @@ description: Notifications are messages that communicate information to the user
 tabs: ["Usage", "Style", "Code", "Accessibility"]
 ---
 
-<ComponentDemo
-  components={[
-    {
-      id: "toast",
-      label: "Toast notification",
-    },
-    {
-      id: "inline",
-      label: "Inline notification",
-    },
-  ]}
->
-  <ComponentVariant
-    id="toast"
-    knobs={{
-      ToastNotification: ["hideCloseButton", "kind", "lowContrast"],
-    }}
-    links={{
-      React:
-        "https://react.carbondesignsystem.com/?path=/story/notifications--toast",
-      Angular:
-        "https://angular.carbondesignsystem.com/?path=/story/components-notification--toast",
-      Vue:
-        "http://vue.carbondesignsystem.com/?path=/story/components-cvtoastnotification--default",
-      Vanilla: "https://the-carbon-components.netlify.com/?nav=notification",
-    }}
-  >{`
-    <div>
-      <ToastNotification
-        caption="00:00:00 AM"
-        iconDescription="describes the close button"
-        subtitle={<span>Subtitle text goes here. <a href="#example">Example link</a></span>}
-        timeout={0}
-        title="Notification title"
-      />
-    </div>
-`}</ComponentVariant>
-  <ComponentVariant
-    id="inline"
-    knobs={{
-      InlineNotification: ["hideCloseButton", "lowContrast"],
-    }}
-    links={{
-      React:
-        "https://react.carbondesignsystem.com/?path=/story/notifications--inline",
-      Angular:
-        "https://angular.carbondesignsystem.com/?path=/story/components-notification--basic",
-      Vue:
-        "http://vue.carbondesignsystem.com/?path=/story/components-cvinlinenotification--default",
-      Vanilla: "https://the-carbon-components.netlify.com/?nav=notification",
-    }}
-  >{`
-  <div>
+<PageDescription>
+
+How to build a notification using React.
```

Comment:
I think this needs to include the link for other frameworks that was added to the template 👀

Reply:
Yes, just this one sentence but no link, so the complete description would be:

> How to build a [component name] using React. For code usage with other frameworks, please follow the links in the live demo on the usage tab.